index.js

```diff
@@ -269,11 +265,11 @@ resolve.sync = function (id, opts) {
 
     opts = opts || {};
     var base = path.dirname(opts.filename);
-    
+
     if (opts.basedir) {
         base = opts.basedir;
     }
-   
+
```

Comment:
Hard to tell from PR, is this a set of spaces or were the spaces removed? Please make sure to remove spaces in empty lines (usually an editor setting).